README.md

```diff
@@ -25,42 +24,129 @@ In the browser, the promise handler for
 [registration.pushManager.subscribe()](https://developer.mozilla.org/en-US/docs/Web/API/PushManager/subscribe)
 returns a
 [PushSubscription](https://developer.mozilla.org/en-US/docs/Web/API/PushSubscription)
-object. This object has a .toJSON() method that will return a JSON
-object that contains all the info we need to encrypt and push data.
+object. This object has a .toJSON() method that will return a JSON object that contains all the info we need to encrypt
+and push data.
 
-As illustration, a subscription info object may look like:
-```
+As illustration, a `subscription_info` object may look like:
+```json
 {"endpoint": "https://updates.push.services.mozilla.com/push/v1/gAA...", "keys": {"auth": "k8J...", "p256dh": "BOr..."}}
 ```
 
 How you send the PushSubscription data to your backend, store it
 referenced to the user who requested it, and recall it when there's
-new a new push subscription update is left as an excerise for the
+new a new push subscription update is left as an exercise for the
 reader.
 
-The data can be any serial content (string, bit array, serialized
-JSON, etc), but be sure that your receiving application is able to
-parse and understand it. (e.g. `data = "Mary had a little lamb."`)
+### Sending Data using `webpush()` One Call
 
-gcm_key is the API key obtained from the Google Developer Console.
-It is only needed if endpoint is
-https://android.googleapis.com/gcm/send
+In many cases, your code will be sending a single message to many
+recipients. There's a "One Call" function which will make things
+easier.
 
-`headers` is a `dict`ionary of additional HTTP header values (e.g.
-[VAPID](https://github.com/mozilla-services/vapid/tree/master/python)
-self identification headers). It is optional and may be omitted.
+```pythonstub
+    from pywebpush import webpush
 
-to send:
+    webpush(subscription_info,
+            data,
+            vapid_private_key="Private Key or File Path[1]",
+            vapid_claims={"sub": "mailto:YourEmailAddress"})
 ```
-WebPusher(subscription_info).send(data, headers)
+This will encode `data`, add the appropriate VAPID auth headers if required and send it to the push server identified
+in the `subscription_info` block.
+
+**Parameters**
+
+*subscription_info* - The `dict` of the subscription info (described above).
+
+*data* - can be any serial content (string, bit array, serialized JSON, etc), but be sure that your receiving
+application is able to parse and understand it. (e.g. `data = "Mary had a little lamb."`)
+
+*vapid_claims* - a `dict` containing the VAPID claims required for authorization (See
```

Comment:
Done. If `aud` is not specified, [it's derived](https://github.com/web-push-libs/pywebpush/pull/45/files#diff-4896b0495b0608d9fbf022a4ed0acc7cR345).